juntagrico/templates/createsubscription/summary.html

```diff
@@ -89,8 +89,17 @@ <h4>{% trans "Anteilscheine" %} <a href="{% url 'cs-shares' %}" class="edit"><i
             <p>{% blocktrans %}Zu je einmalig {{ c_currency }} {{ c_share_price }}{% endblocktrans %}</p>
         {% endif %}
 
+
+
         <form action="" method="post">
             {% csrf_token %}
+            <h4>Kommentar</h4>
+            <div class="form-group row">
+                <div class="col-md-12">
+                    <textarea name="comment" cols="40" rows="3" maxlength="4000" class="textarea form-control"
+                              id="id_comment">{{ main_member.comment }}</textarea>
+                </div>
+            </div>
```

Comment:
Now it might be worth to use a form class (in forms.py) for this and the buttons.

Reply:
moved it to RegisterSummaryForm